src/woo_publications/metadata/management/commands/update_information_categories_from_waardenlijst.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from woo_publications.metadata.update_informatie_category import (
```

Comment:
```suggestion
from ...update_informatie_category import (
```
please use relative paths within the same django app